### PR TITLE
Fix #2817: Add label for ReadingTextSizeActivity

### LINKED ISSUE #2817: [A11y] Add label for ReadingTextSizeActivity
## Task
Introduce label for `ReadingTextSizeActivity` as `Reading Text Size`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by arjunrkaushik**
## Explanation
Fixes #2817: Add label for ReadingTextSizeActivity as Reading Text Size

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -63,7 +63,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.options.ReadingTextSizeActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/reading_text_size"/>
     <activity
```
Comment (prayutsu) [inline app/src/main/AndroidManifest.xml:68]: It requires reformatting.
And also please append `label` to the string id.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt [hunk 2]
```diff
@@ -196,22 +194,22 @@ class AppVersionActivityTest {
   private fun getDateTime(dateTimeTimestamp: Long): String? {
     return oppiaDateTimeFormatter.formatDateFromDateString(
       OppiaDateTimeFormatter.DD_MMM_YYYY,
-      dateTimeTimestamp,
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt:199]: This file isn't related to this PR, revert the changes of this file.

**Review by prayutsu — CHANGES_REQUESTED** [review]
@arjunrkaushik PTAL.

**Comment by prayutsu** [comment]
Also please choose a descriptive branch name and avoid including your name in the branch name. You can look at our [wiki](https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change) to see how to choose a good branch name.

File: app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeActivityTest.kt [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+  private fun setUpTestApplicationComponent() {
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+  }
+
+  private fun createReadingTextSizeActivityIntent(): Intent {
+    return ReadingTextSizeActivity.createReadingTextSizeActivityIntent(
+      ApplicationProvider.getApplicationContext(),
+      READING_TEXT_SIZE,
+      summaryValue = null
```
Comment (arjunrkaushik) [inline app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeActivityTest.kt:100]: @prayutsu What do i pass as the summary value?

Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeActivityTest.kt:100]: @arjunrkaushik you can pass `"Small"`

**Comment by prayutsu** [comment]
@arjunrkaushik Any updates on this?

**Comment by anandwana001** [comment]
Closing this PR as no further update
